sass/nav/_main.scss

```diff
@@ -1,80 +1,178 @@
-.nav-main {
-  @include zindex('navMain');
-  background-color: transparent;
-  transition: background-color 0.2s;
-  font-size: 14px;
-  overflow: hidden;
-  position: fixed;
-  text-transform: uppercase;
-  width: 100%;
-
-  .logo-wrapper {
-    float: left;
-
-    a {
-      display: block;
+@include tablet-large-down() {
+  .nav-main {
+    background: grey;
```

Comment:
you don't use this because at line 7 we have `background-color` which overrides this styles